posixlib/src/main/scala/scala/scalanative/posix/sys/param.scala

```diff
@@ -0,0 +1,9 @@
+package scala.scalanative.posix.sys
+
+import scalanative.unsafe._
+
+@extern
+object param {
+  @name("scalanative_max_path_len")
```

Comment:
Our informal standard would be to make this `scalanative_maxpathlen` so the suffix matches the macro but just in lower case,

Reply:
Thanks!